docs/source/setup.rst

```diff
@@ -29,7 +29,14 @@ Model outputs consist of large csv files that have been designed to work well wi
 
 Installation
 ============
-EnergyPATHWAYS can be installed using the ``environment.yml`` file provided in the repository. This file contains all the necessary packages to run EnergyPATHWAYS. To install the environment, navigate to the directory containing the ``environment.yml`` file and run the following commands::
+First download the EnergyPATHWAYS code from the `EnergyPATHWAYS GitHub repository <https://github.com/EvolvedEnergyResearch/EnergyPATHWAYS>`_ either by downloading the zip file or by cloning the repository using git::
+
+    $ git clone https://github.com/EvolvedEnergyResearch/EnergyPATHWAYS
+
+.. Note::
```

Comment:
Does the EP code run if git is not installed? I thought git was a requirement.

Reply:
Thanks for spotting this!